public/common/TracyTaggedUserlandAddress.hpp

```diff
@@ -0,0 +1,52 @@
+#ifndef __TRACYTAGGEDPTR_HPP__
```

Comment:
This tagged pointer abstraction in the context of the logging feature seems a bit out of place. What is the rationale for it for this branch?